src/components/Button/Button.tsx

```diff
@@ -172,7 +244,7 @@ const Button = forwardRef(
     const mergedRef = useMergeRefs({ refs: [ref, buttonRef] });
 
     const buttonProps = useMemo(() => {
-      return {
+      const props: Record<string, any> = {
```

Comment:
will this work?
```suggestion
      const props: React.HTMLProps<HTMLButtonElement> = {
```

Reply:
Cause some issues on build. because it's not effect on the external API I prefer to keep it like it was and try to change it later (prefer it will not be a blocker)

Reply:
Nope :(. issues with some of the values. prefer it will not be a blocker in this stage.